source/Ship.cpp

```diff
@@ -42,6 +42,7 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 #include <cmath>
 #include <limits>
 #include <sstream>
+#include <iostream>
```

Comment:
This isn't needed.

Reply:
Agreed, this is a left-over from debugging.. I'll remove it.